main.py

```diff
@@ -7,6 +7,6 @@
 faulthandler.enable(open('crash_log.txt', 'w'))
 root = Tk()
 root.wm_title("The Eternal City")
-root.wm_iconbitmap(root, resource_filename('resources.images', 'eternal_logo.ico'))
+#root.wm_iconbitmap(root, resource_filename('resources.images', 'eternal_logo.ico'))
```

Comment:
Didn't mean to commit this, I have it removed locally so it doesn't crash on Ubuntu.